.travis.yml

```diff
@@ -5,9 +5,9 @@ script: bash -ex .travis-ci.sh
 env:
   global:
   - UPDATE_GCC_BINUTILS=1
-  - EXTRA_REMOTES="https://github.com/yomimono/mirage-dev.git#remove-console"
+  - EXTRA_REMOTES="https://github.com/mirage/mirage-dev.git"
```

Comment:
That remote is pointing at `master` of mirage, which this branch isn't; I could have made a separate branch of the remote, though.  I'll do that in the future.